PocketKit/Sources/Analytics/AppEvents/ExpandedSlate.swift

```diff
@@ -30,7 +30,7 @@ public extension Events.ExpandedSlate {
     /**
      Fired when a user clicks a card on Home using the /discover API
      */
-    static func SlateArticleContentOpen(url: String, positionInList: Int, slateId: String, slateRequestId: String, slateExperimentId: String, slateIndex: Int, slateLineupId: String, slateLineupRequestId: String, slateLineupExperimentId: String, recommendationId: String, destination: ContentOpen.Destination) -> ContentOpen {
+    static func SlateArticleContentOpen(url: String, positionInList: Int, recommendationId: String, destination: ContentOpen.Destination) -> ContentOpen {
```

Comment:
since we are at this, can we change the name and use the correct `lowerCamelCase` convention? Same for all the others, thank you!

Reply:
Can we do that in a seperate pr for all the analytics functions? Seems out of place in this one.

Reply:
My point of view is that, if we refactor some code and spot improvements in that same lines of code, then it should be done at the moment. 
The situation in analytics methods is already mixed (meaning we already have methods that use the correct convention, and some others don't), so this won't introduce any disruption.
I can take care of this, but in this case let's leave this PR open and I'll merge it after I operate the changes. Thanks!

Reply:
I agree. My issue with doing it in this pr is if we are going to enforce a standard we should do it via a linter or some pre-established rules. This was just a caught off guard surprise to me.

Reply:
Sounds good, we can merge and then I can fix the naming. One thing: seems like a bunch of tests are failing, can you double check it's not related to the changes?